docs_src/microservices/configuration/CommonEnvironmentVariables.md

```diff
@@ -79,6 +82,9 @@ This environment variable overrides the [`-f/--file` command-line option](../Com
       EDGEX_CONFIG_FILE: "my-config.toml"
     ```
 
+!!! edgey "EdgeX 3.0"
```

Comment:
IMO, this `edgey` is not needed here since this is above the environment variable overrides.
That said, the "`-f/--file` command-line option" above needs to be "`-cf/--configFile` command-line option"

Reply:
Also, all instances of EDGEX_CONFIGURATION_PROVIDER below need to change to `EDGEX_CONFIG_PROVIDER`, except in the `edgey` below.